lib/bearerstrategy.js

```diff
@@ -293,8 +293,9 @@ Strategy.prototype.authenticate = function authenticateStrategy(req) {
 
       var metadataURL = self._options.identityMetadata;
 
-      if (self._options.policyName) {
-        log.info('B2C: using policy %s', self._options.policyName);
+      if (!self._options.policyName) {
```

Comment:
So what I realize I did when I first added B2C was use policyName as the trigger for doing B2C work, so logging that it isn't there isn't correct behavior. Reverted.